presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPruneCountAggregationOverScalar.java

```diff
@@ -84,11 +61,11 @@ public void testFiresOnCountAggregateOverValues()
                                 .step(AggregationNode.Step.SINGLE)
                                 .globalGrouping()
                                 .source(p.values(ImmutableList.of(p.symbol("orderkey")), ImmutableList.of(p.expressions("1"))))))
-                .matches(values(ImmutableMap.of("count_1", 0)));
+                .doesNotFire();
```

Comment:
@kokosing It looks like before this change, the rule was able to remove `count(*)` aggregation over `values`, but now it cannot. Is this intentional?

Reply:
Please see my comment above.